cmd/pitr/recoverer/recoverer.go

```diff
@@ -248,13 +258,16 @@ func (r *Recoverer) recover() (err error) {
 }
 
 func getLastBackupGTID(infoObj io.Reader) (string, error) {
-	sep := []byte("GTID of the last")
-
-	content, err := ioutil.ReadAll(infoObj)
+	content, err := getDecompressedContent(infoObj)
 	if err != nil {
-		return "", errors.Wrap(err, "read object")
+		return "", errors.Wrap(err, "get content")
 	}
 
+	return getGTIDFromContent(content)
+}
+
+func getGTIDFromContent(content []byte) (string, error) {
```

Comment:
`content = content[:e]`
This looks unused

Reply:
removed in https://github.com/percona/percona-xtradb-cluster-operator/pull/743/commits/926bc0586f89de1423004d9559c4f7871a4d2a54 Thanks!